tools/spelling/spelling_dictionary.txt

```diff
@@ -565,6 +565,7 @@ dgst
 dir
 dirname
 djb
+dont
```

Comment:
Why is "dont" added?

Reply:
The comment that has `ContinueAndDontEndStream`. "Dont" is seen as a spelling error.

Reply:
https://github.com/envoyproxy/envoy/pull/13054/files#diff-35f4cb0d915fd79ca0fc7937b5be1d89R365

Reply:
This is a spelling error. "Don't"